src/Acuminator/Acuminator.Tests/Tests/Unit/AttributeInformationTests.cs

```diff
@@ -0,0 +1,286 @@
+﻿using Acuminator.Analyzers;
+using Acuminator.Tests.Helpers;
+using Acuminator.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using PX.Data;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestHelper;
+using Xunit;
+
+namespace Acuminator.Tests
+{
+	
+	public class AttributeInformationTests : DiagnosticVerifier
+	{
+		/* 
+		 *  Tests attribute derived 
+		 * */
+		[Theory]
+		[EmbeddedFileData(@"Dac\PX1030\Unit\AttributeInformationSimpleDacTest.cs")]
+		public void TestAttributeSimpleInformation(string source) =>
+			TestAttributeInformation(source, new List<bool> { false, true, false, false, true, false } );
+
+		[Theory]
+		[EmbeddedFileData(@"Dac\PX1030\Unit\AggregateAttributeInformationTest.cs")]
+		public void TestAggregateAttribute(string source) =>
+			TestAttributeInformation(source, new List<bool> { true, true });
+
+		[Theory]
+		[EmbeddedFileData(@"Dac\PX1030\Unit\AggregateRecursiveAttributeInformationTest.cs")]
+		public void TestAggregateRegursiveAttribute(string source) =>
+			TestAttributeInformation(source, new List<bool> { true, false });
+		
+		private void TestAttributeInformation(string source, List<bool> expected)
+		{
+			Document document = CreateDocument(source);
+			SemanticModel semanticModel = document.GetSemanticModelAsync().Result;
```

Comment:
OK for now, but in the future please use `async` test cases and `await` instead of `Task.Result`